mpv

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+ln -snf "$(xdg-user-dir PICTURES)" "$HOME/.var/app/io.mpv.Mpv/Pictures"
```

Comment:
what is this supposed to achieve?

Reply:
This should be removed. From what I saw, there was no way to screenshot using mpv in the app. The only part is a small popup audio player.

Reply:
Removed